src/main/java/org/opentripplanner/routing/api/request/WheelchairAccessibilityRequest.java

```diff
@@ -1,21 +1,53 @@
 package org.opentripplanner.routing.api.request;
 
+/**
+ * @param slopeExceededReluctance What factor should be given to street edges, which are over the
+ *                                max slope. The penalty is not static but scales with how much you
+ *                                exceed the maximum slope. Set to negative to disable routing on
+ *                                too steep edges.
+ * @param stairsReluctance        Stairs are not completely excluded for wheelchair users but
+ *                                severely punished. This value determines how much they are
+ *                                punished. This should be a very high value as you want to only
+ *                                include stairs as a last result.
+ */
 public record WheelchairAccessibilityRequest(
   boolean enabled,
   WheelchairAccessibilityFeature trips,
-  WheelchairAccessibilityFeature stops
+  WheelchairAccessibilityFeature stops,
+  WheelchairAccessibilityFeature elevators,
+  float inaccessibleStreetReluctance,
+  float maxSlope,
+  float slopeExceededReluctance,
+  float stairsReluctance
```

Comment:
I would like to discuss these in a developer meeting. Here is a few comments:
 - We use a cost(unknown and inaccessible) for trips, stops, and elevators, while we use reluctance for inaccessibleStreet, slopeExceeded and stairs. This seems reasonable, but I would like to be sure.
 - Is there no such thing as `unknownStreetWA` ?
 - Should the street options be wrapped in `StreetWheelchairAccessibilityRequest` - do not have the same abstraction level as trip, and stop. To be picky, it is nor clear what `maxSlope` apply to (is street the only thing a slope may apply to?)
 - Is the `slopeExceededReluctance` enough? For a very short way the extra cost might not be that much, we could add a cost as well?
 - We want to get away from passing Request objects into routers, so should the AStar define an interface for its needed config, and then let this request object implement that. This also apply to the transit part. 
 - Minor: `trips`, `stops` and `elevators` are plural while the type is not a collection - this do not follow the Java code style.

Reply:
There are some valid points and I'm happy to discuss. Nevertheless I will answer a few questions:

- The number of actual streets that have any accessibility information is miniscule. It's very hard to actually find one. Therefore I didn't want to add a "onlyConsiderAccessible" option because when you turn that on you won't ever get any results.
- `StreetWheelchairAccessibilityRequest` is a good idea. 
- maxSlope applies to street edges and to pathways. Theoretically also to things like ramps but they are not modelled yet.

Reply:
- rename stops, trips, elevators to singular
- use doubles instead of floats
- introduce interface to get only the features that you need in StreetEdge